site/src/components/NavbarView/NavbarView.tsx

```diff
@@ -19,41 +23,66 @@ export const Language = {
   users: "Users",
 }
 
-export const NavbarView: React.FC<NavbarViewProps> = ({ user, onSignOut }) => {
+const NavItems: React.FC<{ className?: string; linkClassName?: string }> = ({ className }) => {
   const styles = useStyles()
   const location = useLocation()
+
+  return (
+    <List className={combineClasses([styles.navItems, className])}>
+      <ListItem button className={styles.item}>
+        <NavLink
+          className={combineClasses([styles.link, location.pathname.startsWith("/@") && "active"])}
+          to="/workspaces"
+        >
+          {Language.workspaces}
+        </NavLink>
+      </ListItem>
+      <ListItem button className={styles.item}>
+        <NavLink className={styles.link} to="/templates">
+          {Language.templates}
+        </NavLink>
+      </ListItem>
+      <ListItem button className={styles.item}>
+        <NavLink className={styles.link} to="/users">
+          {Language.users}
+        </NavLink>
+      </ListItem>
+    </List>
+  )
+}
+
+export const NavbarView: React.FC<NavbarViewProps> = ({ user, onSignOut }) => {
+  const styles = useStyles()
+  const [isDrawerOpen, setIsDrawerOpen] = useState(false)
```

Comment:
`nit` Maybe worth a discussion in FE Variety, but I think by convention the setter function need not have the verb like `is` or `has`. For example, I would use `setDrawerOpen` here.

Reply:
I think this would be nice!